xdsl/dialects/riscv.py

```diff
@@ -1396,7 +1513,7 @@ class BltuOp(RsRsOffOperation):
 
 
 @irdl_op_definition
-class BgeuOp(RsRsOffOperation):
+class BgeuOp(RsRsOffOperationInteger):
```

Comment:
This either appears are `...OperationInteger` or `...IntegerOperation`.
Is this on purpose? Shouldn't both be the latter?